### DISCUSSION TITLE
HX711 Waage mit ESP8266 und Tasmota 13.0

### DISCUSSION THREAD

**Discussion by Wilhelm08** [arendst/Tasmota#21546] [Q&A] [answered]
Hilfe,
ich habe eine Waage gebaut mit 4 HX711 Sensoren angeschlossen an das HX711 Board.
ESP 8266 D1 Mini geflasht mit Tasmota13.4.
HX711 angeschlossen an GPIO4: SCK und GPOI5: DAT
Tasmota zeigt als Gewicht 0 kg an.
Wenn ich mich auf der Waage stelle (75kg) zeigt Tasmota 15kg an deshalb muss ich eine  Kalibrierung  duchführen.
Nun mein Problem: Zum Kalibrieren habe ich ein 5kg Gewicht auf der Waage gelegt , in der Konsole "sensor34 2 5000" eingegeben und bestätigt.
Konsole schreibt: 12:05:14.333 RSL: RESULT = {"Sensor34":"Load reference weight"}
20 Sek. später: 12:05:34.391 RSL: RESULT = {"Sensor34":"Calibration failed"}
Warum lässt die Waage nicht Kalibrieren?

**Comment by lalo-uy** [comment, marked as answer]
You need to start calibration with no weight in the scale.
Then put the 5kg when asked by the console.

**Reply by Wilhelm08** [reply]
Danke für die Antwort. So hat es Funktioniert, habe das Referenz Gewicht erhöht auf 75 kg
09:48:56.302 CMD: sensor34 1
09:48:56.306 RSL: RESULT = {"Sensor34":"Reset"}
09:49:05.334 CMD: sensor34 2
09:49:05.339 RSL: RESULT = {"Sensor34":"Load reference weight"}
09:49:23.000 CMD: sensor34 3 75000
09:49:23.006 RSL: RESULT = {"Sensor34":{"Ref":75000,"Cal":120,"Max":20000,"Item":0.0,"Change":"OFF","Delta":0,"Tare":0,"AbsConvA":0,"AbsConvB":0}}
09:49:26.139 RSL: RESULT = {"Sensor34":"Calibrated"}

**Comment by dangod1982-cpu** [comment]
Ich habe auch eine solche Waage, jedoch verliere ich nach einem Neustart alle Einstellungen. Muss dann neu kalibrieren.
